### PR TITLE
Fixed resource adapter test case.

### PR DESCRIPTION
**Pull request by cypo721**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/connector/ResourceAdaptersTestCase.java [hunk 1]
```diff
@@ -115,10 +115,9 @@ public void createProperties() {
         ConfigPropertiesFragment fragment = area.switchToProperty();
         ConfigPropertyWizard wizard = fragment.addProperty();
         Console.withBrowser(browser).waitUntilLoaded();
-
+        wizard.getEditor().text("name", PROPERTY_KEY);
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/connector/ResourceAdaptersTestCase.java:118]: This is sudden inconsistency in property wizard dialog? If this is supposed to be different type of wizard, we should create abstraction for it, if not, we should make sure that there is consistent approach.

**Comment by cypo721** [comment]
Label id = 'name' is used only in this Resource Adapters subsystem. In other subsystems is used label id = 'key' which is used in property wizard.

**Comment by zebrik** [comment]
This inconsistency has been reported as https://issues.jboss.org/browse/HAL-954

**Comment by zebrik** [comment]
Thank you.

**Pull request merged**